### PR TITLE
Update font-source-han-mono

### PR DESCRIPTION
**Pull request by singularitti**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [ ] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [ ] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Created with [`cask-repair`](https://github.com/Homebrew/homebrew-cask/blob/master/CONTRIBUTING.md#updating-a-cask).

### CODE REVIEW

File: Casks/font-source-han-mono.rb [hunk 1]
```diff
@@ -5,6 +5,11 @@
   url "https://github.com/adobe-fonts/source-han-mono/releases/download/#{version}/SourceHanMono.ttc"
   appcast "https://github.com/adobe-fonts/source-han-mono/releases.atom"
   name "Source Han Mono"
+  name "思源等宽"
```
Comment (vitorgalvao) [inline Casks/font-source-han-mono.rb:8]: Are of these necessary? Can’t we trim them down?

Comment (singularitti) [reply Casks/font-source-han-mono.rb:8]: Hi, I just want to provide more information about these fonts. It's a CJK font so I put its name in Chinese, Japanese & Korean there.

Comment (vitorgalvao) [reply Casks/font-source-han-mono.rb:8]: > I put its name in Chinese, Japanese & Korean

That’s three, and I count four. The first and second only differ by a character. Can one of them go without much disruption?

Comment (singularitti) [reply Casks/font-source-han-mono.rb:8]: Oh Yes, one is simplified Chinese & one is traditional Chinese. I can remove the simplified one.

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**